src/Symfony/Component/HttpClient/HttpOptions.php

```diff
@@ -63,9 +63,6 @@ public function setQuery(array $query): static
         return $this;
     }
 
-    /**
-     * @return $this
```

Comment:
this was fine, @welcoMattic had too much :beer: ;)

Reply:
Reverted ;)